code-challenges.test.js

```diff
@@ -10,45 +10,111 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+var colors1 = ["purple", "blue", "green", "yellow", "pink"];
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+var colors2 = [
+  "chartreuse",
+  "indigo",
+  "periwinkle",
+  "ochre",
+  "aquamarine",
+  "saffron",
+];
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
-
-
+describe("chopAndSwap", () => {
+  it("takes in an array, removes the first item from the array and shuffles the remaining content.", () => {
+    expect(chopAndSwap(colors1)).toEqual(["blue", "green", "yellow", "pink"]);
+    expect(
+      chopAndSwap([
+        "chartreuse",
+        "indigo",
+        "periwinkle",
+        "ochre",
+        "aquamarine",
+        "saffron",
+      ]).toEqual(["indigo", "periwinkle", "ochre", "aquamarine", "saffron"])
+    );
+  });
+});
+// I feel like I should revisit this challenge and get a better understanding of jest as my understanding it not any where near w
 // b) Create the function that makes the test pass.
-
-
+const chopAndSwap = (array) => {
+  // used .shift() to remove the first element of the array.
+  arrChop = array.shift();
+};
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
+var nums1 = [3, 56, 90, -8, 0, 23, 6];
 // Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
+var nums2 = [109, 5, 9, 67, 8, 24];
 // Expected output: [5, 109]
 
-
+describe("minMax", () => {
+  it("takes array of numbers and returns array of the minimum and maximum numbers in order.", () => {
+    expect(minMax(nums1)).toEqual([-8, 90]);
+    // I do not understand why the arguments are not executing.
+    expect(minMax(nums2)).toEqual([5, 109]);
+  });
+});
+// Expected1output: [-8, 90]
+const minMax = (array) => {
+  let max = array[0];
+  let min = array[0];
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] > max) {
+      max = array[i];
+    } else if (array[i] < min) {
+      min = array[i];
+    }
+  }
+  let result = ["min: " + min, "max: " + max];
+  return result;
+// I noticed the issue I am having most derives from creating the functions after creating the test.
```

Comment:
You code looks good, I think the issues is that the outcome of your test needs to match EXACTLY the expected output of the test. In this case you are adding some labels in the array on line 78.